File: laba_1_2s/TruckingCompany.py
class TruckingCompany:
    NameofCompany = None
    MoneyforMass = 0
    TotalMass = 0
    def __init__(self, NameofCompany, MoneyforMass, TotalMass):
        self.NameofCompany = NameofCompany
        self.MoneyforMass = MoneyforMass
        self.TotalMass = TotalMass
    def setMoneyforMass(self, MoneyforMass):
        self.MoneyforMass = MoneyforMass
    def setTotalMass(self, TotalMass):
        self.TotalMass = TotalMass
    def getRevenue(self):
        return self.MoneyforMass * self.TotalMass

Gruz = str(input("Введите название компании: "))
MoneyforMass = int(input("Введите количество денег за тонну веса: "))
TotalMass = int(input("Введите количество тонн: "))
Gruz = TruckingCompany(Gruz, MoneyforMass, TotalMass)
print("Компания по перевозке грузов: " + Gruz.NameofCompany +" перевезла " + str(Gruz.TotalMass) + " тонн груза.")
print("За одну тонну груза компания берет " +  str(Gruz.MoneyforMass) + " денег.")
print("Всего компания заработала " +  str(Gruz.getRevenue()) + " денег.")

File: laba_1_2s/src/TruckingCompany.java
public class TruckingCompany {
    String NameofCompany="";
    int MoneyforMass=0 ;
    int TotalMass=0;

    TruckingCompany(String NameofCompany) {
        this.NameofCompany = NameofCompany;
    }
    TruckingCompany(String NameofCompany, int MoneyforMass, int TotalMass) {
        this.NameofCompany = NameofCompany;
        this.MoneyforMass = MoneyforMass;
        this.TotalMass = TotalMass;
    }
//  сетеры
    public void setMoneyforMass(int MoneyforMass) {
        this.MoneyforMass = MoneyforMass;
    }
    public void setTotalMass(int TotalMass) {
        this.TotalMass = TotalMass;
    }
// геттеры
    public int getMoneyforMass() {
        return MoneyforMass;
    }
    public int getTotalMass() {
        return TotalMass;
    }
    public String getNameofCompany() {
        return NameofCompany;
    }
// подсчет прибыли
    public int get_revenue(){
            return MoneyforMass * TotalMass;
    }
}
